pkg/common/runner/service.go

```diff
@@ -10,6 +10,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
+// createSer4vice returns a v1.Service pointing to a given v1.Pod object
```

Comment:
nit: Looks like there was a fat-finger on the 4 key in `createSer4vice`.

Reply:
new keyboard lol